tickets/src/main/java/no/unit/nva/publication/ticket/TicketDtoStatus.java

```diff
@@ -10,6 +10,7 @@ public enum TicketDtoStatus {
     NEW("New"),
     PENDING("Pending"),
     COMPLETED("Completed"),
+    NOT_APPLICABLE("Not Applicable"),
```

Comment:
I will update....

Reply:
"This is esoteric." -> I agree, but we are deep into the implementation of resource service, a tight coupling to dynamoDB and a certain level of domain knowledge is expected of anyone venturing into this part of the code. The tests are the documentation of the functionality. If they don't reveal the code's intent, the tests are poorly written.